projects/cel-cpp/build.sh

```diff
@@ -30,3 +30,21 @@ bazel build -c opt --config=oss-fuzz --linkopt=-lc++ \
 for oss_fuzz_archive in $(find bazel-bin/ -name "*${PACKAGE_SUFFIX}.tar"); do
     tar -xvf "${oss_fuzz_archive}" -C "${OUT}"
 done
+
+if [ "$SANITIZER" = "coverage" ]; then
+    declare -r COVERAGE_SOURCES="${OUT}/proc/self/cwd"
+    mkdir -p "${COVERAGE_SOURCES}"
+    declare -r RSYNC_FILTER_ARGS=(
+        "--include" "*.h"
+        "--include" "*.cc"
+        "--include" "*.hpp"
+        "--include" "*.cpp"
+        "--include" "*.c"
```

Comment:
To be truly generic we might want to include rust, go python etc.

Reply:
The Bazel fuzzing rules currently work with C++ only. Perhaps we can start with C++ and add more suffixes as needed in the future?

Reply:
Got it. Wasn't really suggesting that was needed for this PR.